test/src/test/resources/fixtures/polymorphism/query-with-pass-through-interfaces-and-unions-that-dont-have-typename-in-them-work-as-expected.yml

```diff
@@ -1,5 +1,4 @@
-name: query with pass through interfaces and unions that DONT have __typename in them
-  work as expected
+name: query with pass through interfaces and unions that DONT have __typename in them work as expected
```

Comment:
Couldn't find these tests by project search because the test name was split across lines.